packages/workspace/test/workspace/workspace.test.ts

```diff
@@ -296,13 +307,22 @@ describe('workspace', () => {
       elemMap = getElemMap(await workspace.elements())
     })
 
-    it('should add new elements', () => {
-      expect(elemMap).toHaveProperty(['salesforce.new'])
-    })
     it('should update elements', () => {
-      const lead = elemMap['salesforce.lead'] as ObjectType
-      expect(lead.fields.new_base).toBeDefined()
-      expect(lead.fields.base_field).not.toBeDefined()
+      const salesforceLeadElemID = new ElemID('salesforce', 'lead')
+      const salesforceLeadObject = new ObjectType({ elemID: salesforceLeadElemID })
+      const salesforceText = new ObjectType({ elemID: new ElemID('salesforce', 'text') })
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      salesforceLeadObject.fields.new_base = new Field(salesforceLeadObject, 'new_base', salesforceText)
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      salesforceLeadObject.fields.ext_field = new Field(
+        salesforceLeadObject, 'ext_field', salesforceText, { [CORE_ANNOTATIONS.DEFAULT]: 'foo' }
+      )
```

Comment:
we should not build fields manually
why not create the fields in the `ObjectType` constructor like most other test object types?

```suggestion
      const salesforceText = new ObjectType({ elemID: new ElemID('salesforce', 'text') })
      const salesforceLeadObject = new ObjectType({
        elemID: new ElemID('salesforce', 'lead'),
        fields: {
          new_base: { type: salesforceText },
          ext_field: { type: salesforceText, annotations: { [CORE_ANNOTATIONS.DEFAULT]: 'foo' } },  
        },
      })
```